test/ui/cypress/support/auth.js

```diff
@@ -0,0 +1,51 @@
+import { verifyRequiredEnvVariables } from './../e2e/environment_helpers'
+
+Cypress.Commands.add('loginWithRHAccount', () => {
+   verifyRequiredEnvVariables()
```

Comment:
Let's remove call to verifying environment variables from here and have them added in `cy.before`. That way it is done once per test suite.